crates/sui-indexer-alt-e2e-tests/src/lib.rs

```diff
@@ -298,6 +269,44 @@ impl FullCluster {
     }
 }
 
+pub struct OffchainClusterConfig {
+    pub indexer_args: IndexerArgs,
+    pub consistent_indexer_args: IndexerArgs,
+    pub client_args: ClientArgs,
+    pub indexer_config: IndexerConfig,
+    pub consistent_config: ConsistentConfig,
+    pub jsonrpc_config: JsonRpcConfig,
+    pub graphql_config: GraphQlConfig,
+    pub registry: prometheus::Registry,
+    pub cancel: CancellationToken,
```

Comment:
These should not be path of `OffchainClusterConfig`. Keep the config limited to plain-old data that you can provide defaults for.